src/server/editorServices.ts

```diff
@@ -609,6 +637,7 @@ module ts.server {
                     }
                 }
                 if (content !== undefined) {
+                    content = this.expandTabs(content);

```

Comment:
I think this goes to show that loading the file from disk does not seem to be the right way. Ideally the openfile message to send the contents. the issue is perf when doing the reload, assuming that this is a common occurrence.